### DISCUSSION TITLE
opened Issues in archived Repositories

### DISCUSSION THREAD

**Discussion by BeckerFrank** [eclipse-mylyn/.github#19] [General]
We have the following opened Issues

- org.eclipse.myoyn.commons
  - https://github.com/eclipse-mylyn/org.eclipse.mylyn.commons/issues/2
  - https://github.com/eclipse-mylyn/org.eclipse.mylyn.commons/issues/14
  - https://github.com/eclipse-mylyn/org.eclipse.mylyn.commons/issues/15
  - https://github.com/eclipse-mylyn/org.eclipse.mylyn.commons/issues/16
  - https://github.com/eclipse-mylyn/org.eclipse.mylyn.commons/issues/18
  - https://github.com/eclipse-mylyn/org.eclipse.mylyn.commons/issues/21
  - https://github.com/eclipse-mylyn/org.eclipse.mylyn.commons/issues/23
- org.eclipse.mylyn.context
  - https://github.com/eclipse-mylyn/org.eclipse.mylyn.context/issues/9
- org.eclipse.mylyn.reviews
  - https://github.com/eclipse-mylyn/org.eclipse.mylyn.reviews/issues/5
- org.eclipse.mylyn.builds
  - https://github.com/eclipse-mylyn/org.eclipse.mylyn.builds/issues/8
- org.eclipse.mylyn.tasks
  - https://github.com/eclipse-mylyn/org.eclipse.mylyn.tasks/issues/6
  - https://github.com/eclipse-mylyn/org.eclipse.mylyn.tasks/issues/7
  - https://github.com/eclipse-mylyn/org.eclipse.mylyn.tasks/issues/8
  - https://github.com/eclipse-mylyn/org.eclipse.mylyn.tasks/issues/12

For most of the Issues we can create new Issues in org.eclipse.mylyn.
But what is the best way to handle the following 4 Issues
  - https://github.com/eclipse-mylyn/org.eclipse.mylyn.tasks/issues/7
  - https://github.com/eclipse-mylyn/org.eclipse.mylyn.tasks/issues/8
  - https://github.com/eclipse-mylyn/org.eclipse.mylyn.tasks/issues/12
  - https://github.com/eclipse-mylyn/org.eclipse.mylyn.commons/issues/2

Thougths?

**Comment by ruspl-afed** [comment]
What is the difference between them? Let's
> create new Issues in org.eclipse.mylyn.

with a link to original issues

**Reply by BeckerFrank** [reply]
> What is the difference between them? Let's

The difference is that the creator is not a comitter of this GitHub Orga.
> 
> > create new Issues in org.eclipse.mylyn.
> 
> with a link to original issues

OK, I will then create the new Issues
